lib/linked_in/refresh_token.rb

```diff
@@ -0,0 +1,12 @@
+module LinkedIn
+  # Refresh Token API
+  # @see https://learn.microsoft.com/en-us/linkedin/shared/authentication/programmatic-refresh-tokens
+  class RefreshToken < APIResource
+
+    def refresh_token(options = {})
+      path = 'oauth/v2/accessToken'
```

Comment:
```suggestion
      path = LinkedIn.config.token_url
```

Reply:
@sampatbadhe token_url `/oauth/v2/accessToken` is prepended with a slash, so the url is not formed properly using that. Also, I guess it's better to keep the `token_url` and refresh the token api url separately. Thoughts?